common/src/main/java/com/movie/recommender/common/model/location/LocationMessage.java

```diff
@@ -0,0 +1,23 @@
+package com.movie.recommender.common.model.location;
+
+import lombok.Data;
+import java.io.Serializable;
+
+@Data
+public class LocationMessage implements Serializable {
+    private String message;
```

Comment:
it's not clear why you need message field here
you don't even read its value anywhere in the code